src/node_contextify.cc

```diff
@@ -951,6 +952,125 @@ class ContextifyScript : public BaseObject {
 };
 
 
+void ContextifyContext::CompileFunction(
+    const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Isolate* isolate = env->isolate();
+  Local<Context> context = env->context();
+
+  // Argument 1: source code
+  CHECK(args[0]->IsString());
+  Local<String> code = args[0].As<String>();
+
+  // Argument 2: filename
+  CHECK(args[1]->IsString());
+  Local<String> filename = args[1].As<String>();
+
+  // Argument 3: line offset
+  CHECK(args[2]->IsNumber());
+  Local<Integer> line_offset = args[2].As<Integer>();
```

Comment:
have it been tested these offsets actually do anything for compiling functions?

Reply:
No idea, my guess is that we set them to 0 for like... 99% cases, but I put them in just for parity with `ContextifyScript::New`.

Reply:
well in ContextifyScript the entire source can be offset for dealing with source positions in html bodies and such. if they don't do anything here it would be best to remove them.

Reply:
did you ever verify this?

Reply:
I didn't, sorry.

@addaleax @hashseed is it worth putting these in just for the sake of parity?

Reply:
I think this could be useful.

One way to test this is to throw an exception ans check whether the exception position is affected by the offsets.

Reply:
It indeed works. Will add tests.